components/LibraryCard/LibraryCard.tsx

```diff
@@ -17,16 +17,14 @@ export interface LibraryCardProps {
 }
 
 const LibraryButton = styled(Button)(({ theme }) => ({
-  '&:hover': {
-    backgroundColor: 'white',
-    boxShadow: '0 1px 1px rgba(0, 0, 0, 0.12), 0 2px 2px rgba(0, 0, 0, 0.12), 0 4px 4px rgba(0, 0, 0, 0.12), 0 8px 8px rgba(0, 0, 0, 0.12), 0 16px 16px rgba(0, 0, 0, 0.12)',
-    color: theme.palette.primary.main
-  },
   backgroundColor: 'white',
   borderBottom: 'none',
+  borderRadius: 24,
```

Comment:
The pill shape of the button does not match any other button's style on the site, so let's remove this.

```suggestion
```

Reply:
The site navigation and the "Scroll to Learn More" button have a more circular shape.

Reply:
Only on hover. The header follows that pattern to match the search box. The hero does it to match the header because of proximity. Neither area really "buttons." All elements that look like buttons should follow the Material guidelines as per the MUI `<Button/>` component.